componentes/PainelBuscaAtiva/PainelBuscaAtiva.jsx

```diff
@@ -145,18 +144,42 @@ const PainelBuscaAtiva = ({
             propAgrupamentoImpressao={propAgrupamentoImpressao}
             legendaImpressao={legendaImpressao}
         />
-        Imprimir(1,TabelaImpressaoMounted,lista,aba,sub_aba,trackObject,impressao_com_modal,personalizacao,chavesFiltros,ordenar)
+        const button_choices = ()=>{
+            const button_choices_array = [personalizacao.agrupamento];
+            if(personalizacao.separacaoGrupoPorFolha) button_choices_array.push("equipe_folha_separada");
+            if(personalizacao.ordenacao) button_choices_array.push("ordenar_acs");
+            return button_choices_array;
+        }
+        const filter_applied = [...new Set(chavesFiltros.map(chaveFiltro => Object.keys(chaveFiltro)[0]))]
+        if(impressao_sem_modal) mixpanel.track('button_click', {
```

Comment:
Acho que aqui o `mixpanel` seria `trackObject`, que o nome da prop do componente que recebe o mixpanel

```suggestion
        if(impressao_sem_modal) trackObject.track('button_click', {
```